CONTRIBUTING.md

```diff
@@ -1,188 +1,32 @@
 # Contributing to MicroShift
 
-If you would like to develop MicroShift locally, you can follow this guide on getting
-it installed and running through the provided Makefile.
+Welcome to MicroShift! We are glad you are interested in this project. 
+Pull Requests are always welcome. If you’d like to jump right in with MicroShift, 
+check out the [open issues](https://github.com/redhat-et/microshift/issues). 
+No pull request is too small, we’re pretty sure you might see some typos in the code, 
```

Comment:
Since we're targeting a production-ready state, I think we should avoid language that suggests it might not be there yet.